src/Infrastructure/Service/ValueReceiver/ApiInteractionProcessor.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\InteractionProcessorInterface;
+use SprykerSdk\Sdk\Core\Application\Dto\ReceiverValueInterface;
+use SprykerSdk\Sdk\Infrastructure\Event\RequestDataInjectorInterface;
+use SprykerSdk\Sdk\Infrastructure\Exception\InvalidRequestDataException;
+use SprykerSdk\SdkContracts\Enum\ValueTypeEnum;
+
+class ApiInteractionProcessor implements InteractionProcessorInterface, RequestDataInjectorInterface
```

Comment:
Is it possible to move `ValueReceiver` to the `Infrastructure` folder? We need to get rid of `Service` folder

Reply:
needs to move into another PR. this one already is big.